src/store/modules/todos.js

```diff
@@ -41,8 +111,31 @@ const mutations = {
     },
     deleteTodo: (state, id) => {
         const index = state.todos.findIndex(todo => todo.id === id);
+        console.log('index to delte',index);
+        
         state.todos.splice(index,1);
     },
+    todoCompleted: (state,id) =>{
+        const index = state.todos.findIndex(todo => todo.id === id);
+        console.log('index',index);
+        
+        state.todos[index].comleted = 1;
+    },
+    todoSelected: (state,todo) => {
+        state.selectedTodo = todo;
+    },
```

Comment:
ovo je okej, ali je to lokalno stanje, koristi se samo u jednoj komponenti u jednom momentu
mozes ga ostaviti ovde a mozes i u komponenti pisati